front/app/containers/LandingPage/index.tsx

```diff
@@ -56,8 +58,28 @@ const LandingPage = () => {
 
         <Content>
           <Suspense fallback={<LoadingBox />}>
+            {!isNilOrError(homepageSettings) &&
+              homepageSettings.data.attributes.top_info_section_enabled && (
```

Comment:
The hook isn't merged yet so I think this works for now, once the parent branches have the proper typings and the hook in place we can replace it

Reply:
👍 If we don't have it yet, also log a ticket for it.

Reply:
Good call, on it now